DirectProgramming/Fortran/CombinationalLogic/openmp-primes/sample.json

```diff
@@ -3,23 +3,23 @@
   "name": "OpenMP* Primes",
   "categories": ["Toolkit/oneAPI Direct Programming/Fortran/Combinational Logic"],
   "description": "Fortran Tutorial - Using OpenMP* (OMP)",
-  "toolchain": [ "ifort" ],
+  "toolchain": [ "ifx" ],
   "languages": [ { "fortran": {} } ],
   "targetDevice": [ "CPU" ],
   "os": [ "darwin" ],
```

Comment:
@krisdale Can you remove ``darwin`` from os and add in both ``linux`` and ``windows``?
Ex: ``"os": ["linux", "windows"],``